susemanager-frontend/package.json

```diff
@@ -1,8 +1,13 @@
 {
   "private": true,
   "workspaces": {
-    "packages": ["susemanager-nodejs-sdk-devel", "../web/html/src"],
-    "nohoist": ["**"]
+    "packages": [
+      "susemanager-nodejs-sdk-devel",
+      "../web/html/src"
+    ],
+    "nohoist": [
+      "**"
+    ]
```

Comment:
Is this change relevant to the patch (it doesn't seem to be)?

Reply:
@lneves12 what's your take on this?

Reply:
Addressed :)